File: example/starter.py
import sys

def solve():
  """
  Reads input from stdin, solves the problem, and prints the output.
  """
  try:
    # --- Input reading ---
    n = int(sys.stdin.readline())
    nums = list(map(int, sys.stdin.readline().split()))
    
    # --- Solution ---
    # The following variables are available:
    # n: int
    # nums: list of ints
    
    # TODO: Implement the solution
    
  except (ValueError, IndexError):
    pass

if __name__ == "__main__":
    solve()

File: example/starter.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>

void solve() {
    // --- Input reading ---
    int n;
    std::cin >> n;
    std::vector<long long> nums(n);
    for(int i = 0; i < n; ++i) {
        std::cin >> nums[i];
    }

    // --- Solution ---
    // The following variables are available:
    // n: int
    // nums: std::vector<long long>

    // TODO: Implement the solution
}

int main() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(NULL);
    solve();
    return 0;
}
